### PR TITLE
doc: explain package version rationale

### PR DESCRIPTION
**Pull request by legendecas**
Re: https://github.com/nodejs/node-api-headers/issues/25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,17 @@ in a process to compile and build native addons for Node.js.
 npm i node-api-headers
 ```
 
+<a name="versions"></a>
+
+## Versions
+
+Node-api C headers are backward-compatible. Its version (e.g. `8`) is released
```
Comment (NickNaso) [inline README.md:48]: Node-API

File: README.md [hunk 2]
```diff
@@ -41,6 +41,17 @@ in a process to compile and build native addons for Node.js.
...
 
+<a name="versions"></a>
+
+## Versions
+
+Node-api C headers are backward-compatible. Its version (e.g. `8`) is released 
+separately from the Node.js version stream (e.g. `19.8.1`) and changes are 
+backported to active Node.js LTS lines (e.g. `16.x` and `18.x`).
+
+This package publishes semver-minor versions with new node-api C headers changes.
```
Comment (NickNaso) [inline README.md:52]: Node-API

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by NickNaso** [comment]
@legendecas could you take a look at my comments?

**Comment by legendecas** [comment]
@NickNaso thank you for the suggestions! Fixed.

**Review by NickNaso — APPROVED** [review]

**Pull request merged**